types/react-image-magnify/index.d.ts

```diff
@@ -0,0 +1,229 @@
+// Type definitions for react-image-magnify 2.7
+// Project: https://github.com/gaearon/react-side-effect
+// Definitions by: Sumit Parakh <https://github.com/sumitparakh>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+/// <reference types="react"/>
+
+declare module 'react-image-magnify' {
```

Comment:
This module declaration is not needed.
You can just `export default ReactImageMagnify` directly.
If you want the ability to import each of these interfaces as well, you can export them directly as well.

Reply:
Done.